scene/gui/control.cpp

```diff
@@ -483,6 +483,18 @@ void Control::_validate_property(PropertyInfo &p_property) const {
 		p_property.hint = PROPERTY_HINT_LINK;
 	}
 
+	if (p_property.name == "render_offset") {
+		p_property.hint_string = data.render_offset_relative_to_size
+				? ""
+				: "suffix:px";
+	}
+
+	if (p_property.name == "render_transform_pivot") {
+		p_property.hint_string = data.render_transform_pivot_relative_to_size
+				? ""
+				: "suffix:px";
+	}
```

Comment:
Can't this be shortened to the following?
```suggestion
	if (p_property.name == "render_offset" && !data.render_offset_relative_to_size) {
		p_property.hint_string = "suffix:px";
	}

	if (p_property.name == "render_transform_pivot" && !data.render_transform_pivot_relative_to_size) {
		p_property.hint_string = "suffix:px";
	}
```

Reply:
Wouldn't that prevent it from being set back to "" if one of the `*_relative_to_size` properties is set back to false?

Reply:
Like `_get_property_list()`, `_validate_property()` is called every time properties need to be refreshed (on `notify_property_list_changed()`).
I may be completely misunderstanding it, but that would also mean the `hint_string` is reset back to what is defined in the **PROPERTY_INFO** macro (in this case, nothing).

Still, no deal breaker. Perhaps the current code is even better as is (although the ternary could be in one line)